state/state.go

```diff
@@ -678,15 +677,18 @@ func (i *Indexer) runConsensusFollower(ctx context.Context) {
 		follower.WithBootstrapDir(sporkDir),
 		follower.WithDB(db),
 		follower.WithLogLevel("info"),
-		// NOTE(siva): Enable sync config once it has been merged upstream.
-		// follower.WithSyncCoreConfig(&synchronization.Config{
-		// 	MaxAttempts:   5,
-		// 	MaxRequests:   5,
-		// 	MaxSize:       64,
-		// 	RetryInterval: 4 * time.Second,
-		// 	Tolerance:     10,
-		// }),
+		follower.WithSyncCoreConfig(&synchronization.Config{
+			MaxAttempts:   5,
+			MaxRequests:   5,
+			MaxSize:       64,
+			RetryInterval: 4 * time.Second,
+			Tolerance:     10,
```

Comment:
what is the justification of using 10 for Tolerance?

Reply:
We using the same values as in the `synchronization.DefaultConfig`, except for `MaxRequests` which we bump up to `5`:

* https://github.com/onflow/flow-go/blob/274c6e1688790ea5a928f0cd2d74910cd5d93a37/module/synchronization/core.go#L40